src/const.ts

```diff
@@ -157,7 +190,7 @@ export const CHIP_ERASE_TIMEOUT = 600000; // timeout for full chip erase in ms
 export const MAX_TIMEOUT = CHIP_ERASE_TIMEOUT * 2; // longest any command can run in ms
 export const SYNC_TIMEOUT = 100; // timeout for syncing with bootloader in ms
 export const ERASE_REGION_TIMEOUT_PER_MB = 30000; // timeout (per megabyte) for erasing a region in ms
-export const MEM_END_ROM_TIMEOUT = 50;
+export const MEM_END_ROM_TIMEOUT = 500;
```

Comment:
10x timeout?

Reply:
Had problems. Used the value from https://github.com/adafruit/Adafruit_WebSerial_ESPTool/blob/main/js/esptool.js#L107